### ISSUE TITLE
Retiering of acceleration upgrades for quarry and upping quarry cost

### ISSUE THREAD

**Issue by 0lafe** [GTNewHorizons/NewHorizons#4125]
#### Which modpack version are you using?
2.0.6.8

#### What do you suggest instead/what changes do you propose?
talking to people on discord convinced me that the quarry is still dumb OP, and this can partly be attributed to the speed upgrades. With them being the same tier as the quarry, and not too expensive, it's easy enough to start off with a speed3 version. So new idea is to switch the WA in the speed upgrades to start with LuV, then speed 2 uses ZPM and speed 3 uses UV. Kind of gives an interesting progression line to it. Also maybe switch the gears to tier materials? So osmiridum for 1, naq alloy for 2 and neutronium for 3. 

So, uh also, the quarry isn't terribly expensive. The last change aimed more to tier it better as opposed to making it cost more, which was a bit of a goof on my part. Most of it is fairly cheap with the main cost really just being the LuV field gen, which isn't too appropriate for how good this item is. I suggest changing the recipe from the QED to the ass line, as this is essentially an LuV+ block. 

I'd suggest the recipe being 1 duranium frame box, 2 LuV field gens, 4 LuV robot arms, 2 LuV fluid pumps, 4 LuV pistons, 16x trinium drill (like the thing you would put into a hand held drill), ic2 teleporter, LuV microwave transmitter, 64x ender core, 4x VGa cable. Then 1152L solder, 2000L lube, 576L naquadria, 16000L ender goo. 500s LuV power in the assembly line. This puts it fairly difficultly after mk1 fusion in LuV. 

Considering the last GT ore drill is after this, being post mk2 fusion, it might make sense to tier down the recipes for the multi drills to start at HV and go to LuV pre fusion, when you're then able to make the quarry. Maybe gate them after that tiers rocket, so HV still needs titanium, T2 still needs tungstensteel(mars T2 rocket makes it easier, but needs EV electrolyzer too), T3 needs quantium(venus T4 rocket), and T4 needs mysterious crystal (enceledus T5 rocket). This way the progression of automining can go single block miners for LV-HV pre rocketry, multis starting late HV-early LuV, then fusion gives you the quarry, which is made better each tier after until UHV, when you'll get the void miner from GT++ once that exists. This part is optional, but the upgrades to the drills don't seem to be so great as to put one at mk2 fusion, or mk1. The drilling casings seem to fit this better than their recipes do right now, unless they're intentionally lower tiers, which seems a little weird.

**Comment by ScriptedPiky** [comment]
That actually is a very reasonable and logically expensive change for the Ender Quarry. It still retains the ability to obtain it right after fusion, and still keeps its expensive counterparts that aren't too infeasible, but logical to obtain within that tier. Well done.

The Miners being more easier to obtain (such as, being late HV-LuV where the T4 doesn't need Tritanium/T2 fusion) would make the miners very sought out for, now being more open to the player. Do you have any ideas on what could be used for the miners? Recipe wise that is, but we could fully flesh out the miners

**Comment by 0lafe** [comment]
For the most part this doesn't change much with the T1/2. The T1 right now takes EV motors, a hull and 2 EV circuits. This really just takes titanium, which is kind of the same as what I'm proposing. Similarly with the T2, changing it to EV stuff with tungstensteel puts it roughly in the same spot as right now, except it would take a little less tungstensteel, and not use a little of the fancy rubbers. The change is mostly to not put the T3 past the assembly line, and instead put it mid IV ish after the rocket but before the assembly line. It also means the T4 isn't post mk2 fusion, or mk1, but instead just requires the T5 rocket, which is fairly easy to make in LuV. The drills as a whole aren't great considering their chunkloading requirement, so increasing their range isn't that huge of a buff. I mostly end up running my T2 drill at 3x3 most of the time because of the way chunkloading works for us, even though it has a much larger range. I dont think there's a benefit to the higher tier drills besides their range, but there's a downside of them needing higher tier energy inputs. The T3 drill only takes EV+ and the T4 IV. They're more of a specialized drill you would throw down if you found 2 useful veins near each other, which isn't always the case. I would say its more of a throw and forget, and the higher range just gives you more possibly good veins. Something ending in mid LuV makes sense, considering the prospectors come that tier, so this idea is almost useless by then, and you'd only use a higher range drill if you did find 2 good veins near each other, which isn't too often. 

TL;DR tiering down the controller recipes doesn't really change the T1/2 drills, and the T3/4 aren't really good enough that it makes sense to leave them in LuV/ZPM

**Comment by richardhendricks** [comment]
For the singleblock drills, I plan to add a fortune bonus for the (new) small ores that will be available in the chunk with the orevein.  My idea was to scale it at 2 or 3 per tier, so LV=+2 fortune, HV = +6 fortune.  The multiblock oredrills would also get similar buffs applied, probably +5 per tier (5/10/15).  The fortune buffs would only apply to the small ores/vanilla gems in the chunk.

This would reward people who stop using hammers and start using the miners, and counteracts the loss in density I did making the veins 9 tall. 

I am also thinking of adjusting the processing speed for the drills so that they are slower.  This would encourage setting up transport systems instead of flying back and forth between drills and your base.  I am hesitating because the server vs singleplayer experience here would be a pretty big difference in how it works.

**Comment by 0lafe** [comment]
>counteracts the loss in density I did making the veins 9 tall

Why...

>I am also thinking of adjusting the processing speed for the drills so that they are slower. This would encourage setting up transport systems

What do you mean by this? I got mine just outputting to chests which I fly over to and pull from when it's done, but it's not something I'd benefit in transporting if the speed is slower, it just means I'd go retrieve the ores less often. Also, if you're gonna change them, maybe lower the lube cost? Idk if it changes/tier, but my T2 eats around 128 drums of lube if it mines from ~70 to bedrock, seems a little excessive

**Comment by richardhendricks** [comment]
I lowered the density by 1, and made the veins 2 taller and added small ores throughout the chunk, so that 

 - Mining with a hammer is more convenient now, since the veins are 9 tall.
 - The in-between ores are now similar heights to the primary/secondary ores.
 -  Veins overall are ~ to slightly less "full" than before. About 1/2 of what was lost is replaced with small ores, which can go anywhere in the chunk. Failed small ore placements aren't retried, so overall it's a slight nerf without boosts.
 - Small ores can signal which orevein is present for new players starting out. Makes mountains and cave exploration useful again in a big way. Also makes it faster to figure out what ore is in a center.
 - Small ores are affected by fortune settings, so we have a new balance knob to play with, allowing the automated miners to look more attractive. Also makes putting fortune vs reinforced on your tinker tools a real choice with consequences.
 - Novice players could just ignore the whole 3n+1 thing completely and still have a good experience, since they can see the ores in a chunk based on the small ores.
 - Finally, all ore centers will now be populated. Leaving empty ones was just trying to imitate the results of the old ore algorithm, and after watching it for months, RNGesus can easily result in biased and unfair results (ie, large areas with little/no ores at all). You could still get an area with lots of 1 ore, but at least that would be pretty obvious with the small ores and you can move on sooner. Players who "choose" to stick with manually mining can still do so since across the landscape the density of ores is 33% higher.

> > I am also thinking of adjusting the processing speed for the drills so that they are slower. This would encourage setting up transport systems
> 
> What do you mean by this? I got mine just outputting to chests which I fly over to and pull from when it's done, but it's not something I'd benefit in transporting if the speed is slower, it just means I'd go retrieve the ores less often. 

Yes, hence my reluctance to change it.  Adding all the small ores will make chests fill faster as well since you'll get so much bycatch, don't forget. But I don't know if slowing it down or actually speeding it up would encourage players to automate retrieval, so I will leave it alone.

> Also, if you're gonna change them, maybe lower the lube cost? Idk if it changes/tier, but my T2 eats around 128 drums of lube if it mines from ~70 to bedrock, seems a little excessive

I haven't looked at usage. File a separate ticket.
